README.md

```diff
@@ -1,41 +1,41 @@
-# Answers Headless React
+# Search Headless React
 
-Answers Headless React is the official React UI Bindings layer for [Answers Headless](https://www.npmjs.com/package/@yext/answers-headless).
+Search Headless React is the official React UI Bindings layer for [Search Headless](https://www.npmjs.com/package/@yext/answers-headless).
```

Comment:
I guess the plan is to come back later and update the URL?

Reply:
Yep, a lot of it isn't yet updated because the other packages aren't done yet, including all of the imports from `answers-core` and `answers-headless`.